app/controllers/requests_controller.rb

```diff
@@ -134,4 +135,79 @@ def request_params
     params.require(:request).permit(:name, :cpu_cores, :ram_mb, :storage_mb, :operating_system,
                                     :port, :application_name, :comment, :rejection_information, :status, user_ids: [], sudo_user_ids: [])
   end
+
+  # TODO get credentials and config for currently logged in user
+  # Clones and configures a git repository on dir.
+  def setup_git(dir)
+    # Dir.mkdir(dir) unless File.exists?(dir)
+    uri = "git@github.com:mewitte/PuppetFiles.git"
```

Comment:
Is this "easy to change" for the po? if yes its ok, otherwise only devs can change it.

Reply:
The canonical way of solving this would be a .env file, See https://rubygems.org/gems/dotenv/versions/2.1.0?locale=de